### PR TITLE
Update AutoHeal.java and Commands.java

### PR DESCRIPTION
**Pull request by qloha**
## Summary by CodeRabbit

- **New Features**
	- Enhanced the `AutoHeal` module for improved healing item selection with new settings for `goldenHead`, `soup`, and `goldenHeadName`.
	- Introduced a new command `silenceirc` for setting a `qqId` in the `SilenceIRC` module.

- **Bug Fixes**
	- Improved case-insensitivity for module name matching in command handling.

These updates streamline healing item management and enhance command functionality for a better user experience.

### CODE REVIEW

File: src/main/java/keystrokesmod/module/impl/player/AutoHeal.java [hunk 1]
```diff
@@ -42,15 +45,30 @@ public void onRender(TickEvent.RenderTickEvent event) {
...
+                                    toSlot = slot;
+                                    break;
+                                }
+                            } else {
+                                toSlot = slot;
+                                break;
+                            }
+                        }
+                    }
+                }
```
[the remark this answers is not in the document]
Comment (qloha) [reply src/main/java/keystrokesmod/module/impl/player/AutoHeal.java:67]: no, its only for the hotbar
thanks for the suggestion though

File: src/main/java/keystrokesmod/utility/Commands.java [hunk 2]
```diff
@@ -497,6 +500,19 @@ public static void rCMD(@NotNull String c) {
                     }
                     print("&cInvalid profile.", 1);
                 }
+            } else if (firstArg.equals("silenceirc")) {
```
Comment (xia-mc) [inline src/main/java/keystrokesmod/utility/Commands.java:503]: I have remove this module, so don't add it back

**Review by xia-mc — CHANGES_REQUESTED** [review]
Don't add 'SilenceFix' command to commands

File: src/main/java/keystrokesmod/utility/Commands.java (hunk 2 above)
Comment (qloha) [reply src/main/java/keystrokesmod/utility/Commands.java:503]: 👍

**Comment by xia-mc** [comment]
Manual merged

**Comment by qloha** [comment]
<3

**Pull request closed without merging**